contracts/utils/SpigotLib.sol

```diff
@@ -213,8 +222,8 @@ library SpigotLib {
     }
 
     /** see Spigot.getEscrowed*/
-    function getEscrowed(SpigotState storage self, address token) external view returns (uint256) {
```

Comment:
i dont see `getOperatorTokens` here

Reply:
actually we dont even use this function in Spigot.sol. you can delete it

Reply:
this must be stale bc i dont have this func on my local branch